test/src/api/commands/web-element/testGetProperty.js

```diff
@@ -109,4 +109,53 @@ describe('element().getProperty() command', function () {
     const resultValue = await resultPromise.value;
     assert.deepStrictEqual(resultValue, ['.signup']);
   });
+
+  it('test .element.find().prop()', async function() {
+    MockServer.addMock({
+      url: '/session/13521-10219-202/element/0/property/classList',
+      method: 'GET',
+      response: JSON.stringify({
+        value: ['.signup']
+      })
+    }, true);
+
+    const resultPromise = this.client.api.element.find('#signupSection').prop('classList');
+    assert.strictEqual(resultPromise instanceof Element, false);
+    assert.strictEqual(typeof resultPromise.find, 'undefined');
+
+    assert.strictEqual(resultPromise instanceof Promise, false);
+    assert.strictEqual(typeof resultPromise.then, 'function');
+
+    const result = await resultPromise;
+    assert.strictEqual(result instanceof WebElement, false);
+    assert.deepStrictEqual(result, ['.signup']);
+
+    const resultValue = await resultPromise.value;
+    assert.deepStrictEqual(resultValue, ['.signup']);
+  });
+
+  it('test .element().find().property()', async function(){
```

Comment:
Test case for "test .element().property()" is already present as the second test case in this test suite, you just need to add a third test case for the `.prop()` alias with similar contents.